### PR TITLE
update docs clarifying how the chartpath should be used when the gitr…

### PR DESCRIPTION
**Pull request by bluehawk27**
…epo artifact is used

### CODE REVIEW

File: content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md
Comment (dbyron-sf) [inline content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md:62]: Is this really correct?  For `root/helm/Chart.yaml` woudn't helmChartFilePath be `root/helm` ?  Maybe it's the use of `root` that's throwing me off.  For me, if a path starts with a slash, that slash means root...So maybe this becomes...
```suggestion
  Example: If Chart.yaml is in a directory named helm  (i,e.`/helm/Chart.yaml`) then `helmChartFilePath` would be `/helm`
```
but is the presence (or absence) of the leading slash the thing that was causing you trouble?

Comment (jasonmcintosh) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md:62]: Hrmm, how about this:
Example
```
Give a git repo where your Chart.yaml is in:
  sub/folder/Chart.yml

```
then you'd want:
```
helmChartFilePath: "sub/folder/"
```

Comment (dbyron-sf) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md:62]: Sure.  Does a leading slash break things?

Comment (bluehawk27) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md:62]: @dbyron-sf I agree with on that. I'm afraid someone would actually use the leading slash in the file path, which actualy doesnt work.

Comment (dbyron-sf) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md:62]: So perhaps include a note about "no leading slash" ...

**Review by jasonmcintosh — APPROVED** [review]

File: content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md
Comment (jasonmcintosh) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md]: Tossed my approval in now for what it's worth ;) (I don't have real approval permissions :P )

**Review by dbyron-sf — APPROVED** [review]
also not an approver, but lgtm.  Perhaps @german-muzquiz can give this a look?

**Pull request merged**
